### PR TITLE
Add remediation steps

### PR DESCRIPTION
**Pull request by gh-andrem**
To better remediate WindowsEventLogReader error

### CODE REVIEW

**Comment by dcurwin** [comment]
@gh-andrem - Can I ask if you are an MS employee or how you otherwise came to these conclusions?

**Comment by dcurwin** [comment]
@gh-andrem  - Ok, I'll check with the product team and see about integrating it into the documentation. Thanks!

**Comment by Court72** [comment]
@dcurwin

Thanks for reviewing the proposed changes.

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by Court72** [comment]
Hi @dcurwin, now that the PR is triaged, the review team won't see it again until you've signed off.

File: ATPDocs/troubleshooting-known-issues.md [hunk 1]
```diff
@@ -336,10 +336,20 @@ A Discretionary Access Control List is limiting access to the required event log
...
 
 `(A;;0x1;;;S-1-5-80-818380073-2995186456-1411405591-3990468014-3617507088)`
 
+Check if the DACL for the Security Event Log was configured by a GPO:
+
+`Policies > Administrative Templates > Windows Components > Event Log Service > Security > Configure log access`
+
+Append the entry above to the existing policy. Run `wevtutil gl security` after to verify that the entry was added.
+
+The local MDI logs should now display:
```
Comment (dcurwin) [inline ATPDocs/troubleshooting-known-issues.md:349]:
```suggestion
The local Defender for Identity logs should now display:
```

File: ATPDocs/troubleshooting-known-issues.md [hunk 2]
```diff
@@ -336,10 +336,20 @@ A Discretionary Access Control List is limiting access to the required event log
...
-Ensure that the Discretionary Access Control List includes the following entry:
+Ensure that the Discretionary Access Control List (DACL) includes the following entry (this is the SID of the AATPSensor service).
 
 `(A;;0x1;;;S-1-5-80-818380073-2995186456-1411405591-3990468014-3617507088)`
 
+Check if the DACL for the Security Event Log was configured by a GPO:
+
+`Policies > Administrative Templates > Windows Components > Event Log Service > Security > Configure log access`
+
+Append the entry above to the existing policy. Run `wevtutil gl security` after to verify that the entry was added.
```
Comment (dcurwin) [inline ATPDocs/troubleshooting-known-issues.md:347]:
```suggestion
Append the entry above to the existing policy. Run `C:\Windows\System32\wevtutil.exe gl security` afterwards to verify that the entry was added.
```

**Comment by dcurwin** [comment]
@gh-andrem  - Thanks for your contribution. It will be published shortly.

**Pull request merged**